### PR TITLE
18 2nd weapon

### PR DESCRIPTION
**Pull request by ofirschwartz2**

### CODE REVIEW

File: Assets/Scripts/Shot2Script.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+using Assets.Common;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Shot2Script : MonoBehaviour
+{
+    public bool UseConfigFile;
+    public float Lifetime;
+    private float endOfLiveTime = 0;
```
Comment (AmirIzh) [inline Assets/Scripts/Shot2Script.cs:10]: private fields starts with _

File: Assets/Scripts/gunController.cs [hunk 2]
```diff
@@ -7,18 +7,24 @@ public class gunController : MonoBehaviour
...
     public Transform firePoint;
-    public GameObject shotPrefab;
+    public List<GameObject> shotsPrefab;
+    private int currentWeapon = 0;
+    private bool canSwitch = true;
+    private GameObject shotPrefab;
     // TODO: the gun should not hold it's parent (halalit), it's a bad practice, it only need it's x and y positions, so it should use a function of the halalitMovementController that will get them.
     public GameObject halalit;
     public Joystick gunJoystick;
+    public Joystick weaponSwitchJoystick;
```
Comment (AmirIzh) [inline Assets/Scripts/gunController.cs:23]: for all of the fields above, public fields starts with a capital letter

**Review by AmirIzh — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Shot2Script.cs (hunk 1 above)
Comment (ofirschwartz2) [reply Assets/Scripts/Shot2Script.cs:10]: done

File: Assets/Scripts/gunController.cs (hunk 2 above)
Comment (ofirschwartz2) [reply Assets/Scripts/gunController.cs:23]: done

**Review by AmirIzh — APPROVED** [review]

**Pull request merged**